services/web/server/src/simcore_service_webserver/resource_manager/garbage_collector.py

```diff
@@ -16,6 +16,9 @@
 
 from .config import APP_GARBAGE_COLLECTOR_KEY, get_garbage_collector_interval
 from .registry import RedisResourceRegistry, get_registry
+from simcore_service_webserver.projects.projects_api import delete_project_from_db
+from simcore_service_webserver.login.cfg import get_storage
+from simcore_postgres_database.models.users import UserRole
```

Comment:
;) that might be c++ich but then I did not have to include the modules... just kidding...

but the idea is to keep this file as independent as possible for when we move this to an independent service. that is why I used the observer pattern.

Reply:
👍 for loose coupling. will also enforce this policy now